drake/examples/Pendulum/runDynamics.cpp

```diff
@@ -0,0 +1,20 @@
+
+#include <iostream>
+#include "Pendulum.h"
+#include "LCMCoordinateFrame.h"
+
+using namespace std;
+
+int main(int argc, char* argv[]) {
+  DrakeSystemPtr p = make_shared<Pendulum>();
```

Comment:
Why allocate on the heap? It's not super wrong or anything, it's just unnecessary in my opinion.